package.json

```diff
@@ -47,7 +47,7 @@
     "opener": "^1.5.2",
     "p-limit": "^3.1.0",
     "patch-package": "^6.4.7",
-    "posthog-node": "^2.4.0",
+    "posthog-node": "^2.5.1",
```

Comment:
This pkg requires node 15+which is kind of weird. I guess we have the ci tests for node 14.

Reply:
That's odd. The tests seem to have passed on node 14 and npm didn't alert me of the issue. How did you find out about the requirement?

Reply:
Was looking at the package-lock.json trying to figure out why my previous patch didn't work. Thought maybe it was using 2.5+ in the package-lock which would explain why the override wasn't working anymore.